app/grandchallenge/archives/static/archives/js/archive_item_update.js

```diff
@@ -0,0 +1,10 @@
+$('#ajaxDataTable').on( 'init.dt', function() {

```

Comment:
Can you do this without jquery?

Reply:
hm, I'm not sure how actually. I went with what is recommended in the datatables documentation. I also tried adding an event listener to the data table, but couldn't get that work. How else would I do it?

Reply:
Ah, I neglected that this is datatables, then indeed not possible, sorry my fault!